acceptance/suites/tests/555-puppetserver-service/reboot.rb

```diff
@@ -0,0 +1,63 @@
+if (options[:type] == 'pe') then
+  pssn='pe-puppetserver'
+else
+  pssn='puppetserver'
+end
+
+test_name "Validates the ability of the #{pssn} service to reliably start after a reboot."
+
+max=10
+sleeptime=40
+retryCounter=0
+step "\n-----------\nValidate #{pssn} is up and running, then reboot, repeat #{max} times\n-----------"
+
+for i in 1..max 
```

Comment:
Disclaimer: I am not a QA 

Ahh ok, that makes sense to me now. But hmm, do you think running this loop 10 times is a bit excessive? It seems like if it comes up once after a reboot then it should be working.